src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BaseWindow.cs

```diff
@@ -54,11 +54,16 @@ public virtual void Finish(bool result)
             RaiseOnClose(result);
         }
 
-        protected void RaiseOnClose(bool result)
+        protected virtual void RaiseOnClose(bool result)
```

Comment:
TL;DR: `ClearOnClose` should be called directly from `RaiseOnClose` so that subclasses don't accidentally forget to do it. That means this method doesn't need to be virtual.